assets/js/translator.js

```diff
@@ -460,6 +460,11 @@ function populateTranslationList() {
         numDstCols = Math.ceil(dstLangs.length / TRANSLATION_LIST_ROWS) < (TRANSLATION_LIST_COLUMNS + 1)
             ? Math.ceil(dstLangs.length / TRANSLATION_LIST_ROWS)
             : TRANSLATION_LIST_COLUMNS;
+
+    var thresholdWindowWidth = 1020;
```

Comment:
Move this to a constant. Are you sure it's not `1024`? That would make much mores sense since that's where Bootstrap defines break points as well.